virttest/utils_libvirt/libvirt_nested.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+from avocado.core import exceptions
+from avocado.utils import process
+
+from virttest import utils_libvirtd
+from virttest import utils_package
+
+
+def install_virt_pkgs(vm_session):
+    """
+    Install required virt packages for nested virt test
+
+    :param vm_session: session to the vm
+    :raises: exceptions.TestError if installation fails
+    """
+    pkg_names = ['libvirt', 'qemu-kvm']
+    logging.debug("Virt packages will be installed")
+    pkg_mgr = utils_package.package_manager(vm_session, pkg_names)
+    if not pkg_mgr.install():
+        raise exceptions.TestError("Package '%s' installation "
+                                   "fails" % pkg_names)
+    libvirtd = utils_libvirtd.Libvirtd()
+    libvirtd.restart()
+
+
+def enable_nested_virt_on_host():
+    """
+    Enable nested virt on the host
+
+    :raises: exceptions.TestError if nested virt is not set successfully
+    """
+    cmd = 'modprobe -r kvm_intel; modprobe -r kvm; modprobe kvm nested=1; modprobe kvm_intel nested=1'
```

Comment:
Do we need to consider AMD as well?

Reply:
@Yingshun , confirmed with feature owner, only Intel is needed to be considered. I updated.